alot/commands/search.py

```diff
@@ -135,10 +135,9 @@ def apply(self, ui):
                           'default': 'True',
                           'help': 'postpone a writeout to the index'}),
         (['tags'], {'help': 'comma separated list of tags'})],
-    help='flip presence of tags on this thread. A tag is considered present '
-         'if at least one message contained in this thread is tagged with it. '
-         'In that case this command will remove the tag from every message in '
-         'the thread.')
+    help='flip presence of tags on this thread: a tag is considered present '
+         'and removed if at least one message in this thread is tagged with '
```

Comment:
you're missing `will be` between `and` and `removed`, or some other change to make this sentence grammatically correct.

Reply:
I don’t believe I am, unless you insist on the future tense: “a tag (is (considered present) and (removed)) if ...”.

Reply:
I asked a co-worker to look at this to make sure I'm not way off, but both of us parsed this as "flip present of tags on this thread: a tag is considered (present and removed) if at least one...". We both agree that the original message reads more naturally.

Reply:
”Will be removed“ it is then. I do tend to overcomplicate my sentences sometimes. BTW, the reason for changing it was to turn the whole thing into a single sentence (as it must be if we’re not capitalizing it).